Makefile

```
@@ -96,18 +96,104 @@ release: validate test-install
+_print-scripts-spex:
+	@echo $(SCRIPTS_spex)
+_print-scripts-spex-gates:
+	@echo $(SCRIPTS_spex_gates)
+_print-scripts-spex-collab:
+	@echo $(SCRIPTS_spex_collab)
+_print-scripts-spex-deep-review:
+	@echo $(SCRIPTS_spex_deep_review)
+_print-scripts-spex-detach:
+	@echo $(SCRIPTS_spex_detach)
+
+sync-scripts-check:
```

checkmake (0.3.2)

[warning] Target body for "sync-scripts-check" exceeds allowed length of 5 lines (43).

(maxbodylength)